tensorflow_lite_support/python/task/processor/proto/qa_answers_pb2.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Question answers protobuf."""
+
+import dataclasses
+from typing import Any, List
+
+from tensorflow_lite_support.cc.task.processor.proto import qa_answers_pb2
+from tensorflow_lite_support.python.task.core.optional_dependencies import doc_controls
+
+
+_PosProto = qa_answers_pb2.QaAnswer.Pos
+_QaAnswerProto = qa_answers_pb2.QaAnswer
+_QuestionAnswererResultProto = qa_answers_pb2.QuestionAnswererResult
+
+
+@dataclasses.dataclass
+class Pos:
+  """Represents the logit and offset of the answer related to context.
+
+  Attributes:
+    start: The start offset index of the answer.
+    end: The end offset index of the answer.
+    logit: The logit of the answer.
```

Comment:
Change to `logit: The logit of the answer. Higher value means it's more likely to be the answer.`